README.md

```diff
@@ -1,4 +1,5 @@
 # Portfolio-setup-and-mobile-first
+
```

Comment:
````
<div align="center">
  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
  <br/>
  <h3><b>Portfolio-setup-and-mobile-first</b></h3>
</div>

````
- [ ] Kindly make sure you provided a correct link to your logo because it is broken or if don't have a logo it you may delete this section to make your README more professional 👍 
![logo](https://user-images.githubusercontent.com/57408419/227231553-a7c8fc02-2dad-4b07-9662-3926df487ca2.JPG)